diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index 898d9672187ca1aee9f65119b412dba727675ec1..2de46302aa288df14be908431e068452126e4fbe 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -109,6 +109,7 @@ class AvatarToolbarButton : public ToolbarButton {
   void NotifyShowEnterpriseTextEndedForTesting() const;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
