diff --git a/chrome/elevation_service/BUILD.gn b/chrome/elevation_service/BUILD.gn
index ba0910251d9447573a9b4d34ee7e9863d7ff3c14..e4d99d156f352eecc5775170deae565b33ab7330 100644
--- a/chrome/elevation_service/BUILD.gn
+++ b/chrome/elevation_service/BUILD.gn
@@ -10,6 +10,7 @@ import("//testing/test.gni")
 
 midl("elevation_service_idl") {
   sources = [ "elevation_service_idl.idl" ]
+  sources -= [ "elevation_service_idl.idl" ] sources += ["//brave/chromium_src/chrome/elevation_service/elevation_service_idl.idl" ]
 
   writes_tlb = true
 }
@@ -98,6 +99,7 @@ source_set("lib") {
     "crypt32.lib",
     "rpcrt4.lib",
   ]
+  deps += brave_elevation_service_lib_deps
 }
 
 process_version_rc_template("version_resources") {
