diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7a4534d39b49196759bac51052e9bbe6ab3a4f48..a1841fe6509f4a7c1f378bc3ab1000c0759311d7 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2700,6 +2700,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
