diff --git a/chrome/browser/ui/views/bubble/webui_bubble_manager.h b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
index 5a572711eda1eaf9fc8c8aebbfd0dec9d5e5ef66..8d4f787c05caeb6d63035471186a23738e7540e3 100644
--- a/chrome/browser/ui/views/bubble/webui_bubble_manager.h
+++ b/chrome/browser/ui/views/bubble/webui_bubble_manager.h
@@ -209,6 +209,7 @@ class WebUIBubbleManagerT : public WebUIBubbleManager {
 
     auto bubble_view = std::make_unique<WebUIBubbleDialogView>(
         anchor_view_, contents_wrapper->GetWeakPtr(), anchor, arrow);
+    BRAVE_WEBUI_BUBBLE_MANAGER_T_CREATE_WEB_UI_BUBBLE_DIALOG
 
     // Register callback to emit histogram when the widget is created
     if (bubble_init_start_time_) {
