diff --git a/chrome/app/BUILD.gn b/chrome/app/BUILD.gn
index 3b0a3ea89a6528816946a372e83fc1712b4a04d6..ca18d8094702d5b1b8c1dd77f7293b8ae1977da7 100644
--- a/chrome/app/BUILD.gn
+++ b/chrome/app/BUILD.gn
@@ -107,6 +107,7 @@ grit("generated_resources") {
   if (is_android) {
     outputs += android_generated_java_resources
   }
+  deps = [ "//brave/app:brave_generated_resources_grit" ]
 }
 
 if (is_android) {
