diff --git a/chrome/browser/external_protocol/external_protocol_handler.cc b/chrome/browser/external_protocol/external_protocol_handler.cc
index bad6ac790ae1f8f0e1d2dfa4bd4ac2412fe43a6a..08464dc380a447ca136348708cfa6c002324f80f 100644
--- a/chrome/browser/external_protocol/external_protocol_handler.cc
+++ b/chrome/browser/external_protocol/external_protocol_handler.cc
@@ -362,7 +362,7 @@ ExternalProtocolHandler::BlockState ExternalProtocolHandler::GetBlockState(
   if (scheme == "mailto") {
     base::UmaHistogramEnumeration(kBlockStateMetric,
                                   BlockStateMetric::kAllowedDefaultMail);
-    return DONT_BLOCK;
+    return UNKNOWN;
   }
 
   PrefService* profile_prefs = profile->GetPrefs();
