diff --git a/components/content_settings/core/common/cookie_settings_base.cc b/components/content_settings/core/common/cookie_settings_base.cc
index 6828598ce4ed1a49b1135bc06a75d5bf7fedee7d..5283856db97820946e21bab082026eedef56b5d2 100644
--- a/components/content_settings/core/common/cookie_settings_base.cc
+++ b/components/content_settings/core/common/cookie_settings_base.cc
@@ -512,7 +512,7 @@ CookieSettingsBase::DecideAccess(const GURL& url,
   if (!IsAllowed(setting)) {
     return BlockAllCookies{};
   }
-
+  BRAVE_COOKIE_SETTINGS_BASE_DECIDE_ACCESS
   if (!ShouldBlockThirdPartyCookies() &&
       !Are3pcsForceDisabledByOverride(overrides)) {
     return AllowAllCookies{
@@ -610,6 +610,7 @@ CookieSettingsBase::GetCookieSettingInternal(
   const ContentSetting cookie_setting = GetContentSetting(
       url, first_party_url, ContentSettingsType::COOKIES, &setting_info);
 
+  BRAVE_COOKIE_SETTINGS_BASE_GET_COOKIES_SETTINGS_INTERNAL
   const bool is_explicit_setting =
       !setting_info.primary_pattern.MatchesAllHosts() ||
       !setting_info.secondary_pattern.MatchesAllHosts();
