diff --git a/components/strings/BUILD.gn b/components/strings/BUILD.gn
index 5dd1ab8510602aa95a9b3983d4d36165392bda00..3ecfe4ee575c98c3253ffacf20688256cf0135ae 100644
--- a/components/strings/BUILD.gn
+++ b/components/strings/BUILD.gn
@@ -51,6 +51,7 @@ grit("components_strings") {
   if (is_android) {
     outputs += android_components_strings_java_resources
   }
+  deps = [ "//brave/components/resources:strings" ]
 }
 
 if (is_android) {
