diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index 822d6dc3951e25d54f3b2f16ac415c8d79981d73..84c9c934dbe6a3d889fe6fa9deaabfd3702457f6 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -365,6 +365,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
