diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 9bcd71abb6fa2074f6620459ee715a40d812cbf3..4f1607518d426a96afcf2f0c9c30de8640d59993 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -264,6 +264,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
