diff --git a/components/permissions/request_type.cc b/components/permissions/request_type.cc
index fad9b73fa4fef5db944ff804131afb0693c192dd..7eb14460e9e8192215c5dfee5b248311811f15bf 100644
--- a/components/permissions/request_type.cc
+++ b/components/permissions/request_type.cc
@@ -460,6 +460,7 @@ const char* PermissionKeyForRequestType(permissions::RequestType request_type) {
         return "window_management";
       }
 #endif
+    BRAVE_PERMISSION_KEY_FOR_REQUEST_TYPE
   }
 
   return nullptr;
