diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 0465aaa8fc6949fd8c6fef30391b197661381756..779046759958d6faf086526b4aabc7d9557ff253 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -882,6 +882,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
