diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index c4057f9e97a2cab7a824f5614316f06aadf33514..3e6182de4d1fdbab5230e75e5fdb1d95314eddda 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -4743,6 +4743,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame())
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
 
