diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index fdc1defcdd260872d4e5ce512dfea3bad48bb9e3..70b55e1aa0247729d4557b2f41641a5bbf5e93da 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
