diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index dd289563cc6c63817a6cee059385950779aa7215..958c832a19e95ad0c909f90cb21a1ae479857c4e 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -477,6 +477,7 @@ bool LocalDOMWindow::CanExecuteScripts(
   bool allow_script_content_setting =
       GetFrame()->GetContentSettings()->allow_script;
   bool script_enabled = allow_script_renderer && allow_script_content_setting;
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
