diff --git a/third_party/blink/renderer/modules/scheduler/scheduled_action.cc b/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
index 3955cd6f2b42f608db47f45cf0d9f6a3c3518de3..7985d6a189563a8b81d8a62dad5f1ebe3157c450 100644
--- a/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
+++ b/third_party/blink/renderer/modules/scheduler/scheduled_action.cc
@@ -90,6 +90,7 @@ ScheduledAction::ScheduledAction(ScriptState* script_state,
   } else {
     UseCounter::Count(target, WebFeature::kScheduledActionIgnored);
   }
+  BRAVE_SCHEDULED_ACTION_STRING_HANDLER_CONSTRUCTOR
 }
 
 ScheduledAction::~ScheduledAction() {
