diff --git a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
index 307171c7ee627bebb2ccf3eabc0a96e9768cad5a..a1fdf7cced402eec3280d709e227ac91f4c9df04 100644
--- a/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
+++ b/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc
@@ -2324,6 +2324,7 @@ ImageData* BaseRenderingContext2D::getImageData(
 }
 
 ImageData* BaseRenderingContext2D::getImageDataInternal(
+    BRAVE_GET_IMAGE_DATA_PARAMS
     int sx,
     int sy,
     int sw,
@@ -2489,6 +2490,7 @@ ImageData* BaseRenderingContext2D::getImageDataInternal(
       DCHECK(!bounds.intersect(SkIRect::MakeXYWH(sx, sy, sw, sh)));
     }
   }
+  BRAVE_GET_IMAGE_DATA
 
   return image_data;
 }
@@ -3158,6 +3160,7 @@ TextMetrics* BaseRenderingContext2D::measureText(const String& text) {
       return MakeGarbageCollected<TextMetrics>();
     }
 
+    BRAVE_BASE_RENDERING_CONTEXT_2D_MEASURE_TEXT
     canvas->GetDocument().UpdateStyleAndLayoutTreeForElement(
         canvas, DocumentUpdateReason::kCanvas);
   }
