diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index b4a959a7d99449a083166dc7de4250133badf862..86dd98bcac1b97631c6804217867de8a0e4b210c 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -476,6 +476,7 @@ bool LocalDOMWindow::CanExecuteScripts(
   bool allow_script_content_setting =
       GetFrame()->GetContentSettings()->allow_script;
   bool script_enabled = allow_script_renderer && allow_script_content_setting;
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
