diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
index b5fd10dd6f0b1e2fe890baf9ed149f8404dd62e4..452ab828af6160c43765f7de31d7aaffa22533b1 100644
--- a/tools/typescript/ts_library.gni
+++ b/tools/typescript/ts_library.gni
@@ -81,6 +81,7 @@ template("ts_library") {
     }
 
     if (defined(path_mappings)) {
+      import("//brave/tools/typescript/sources.gni") foreach(mapping, brave_path_mappings) { path_mappings += [ mapping[0] + rebase_path(mapping[1], target_gen_dir) ] }
       args += [ "--path_mappings" ] + path_mappings
     }
 
