diff --git a/ui/webui/resources/tools/build_webui.gni b/ui/webui/resources/tools/build_webui.gni
index 632e4e6014bfaf77de5b736d5b2dc29bbda9c0f3..4c24c49f1978b0f351eba0d7837f17160bfd60cb 100644
--- a/ui/webui/resources/tools/build_webui.gni
+++ b/ui/webui/resources/tools/build_webui.gni
@@ -185,6 +185,7 @@ template("build_webui") {
       enable_removal_comments = enable_source_maps
     }
     in_files = ts_files
+    if (defined(invoker.exclude_ts_preprocess_files)) { in_files -= invoker.exclude_ts_preprocess_files } if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
   }
 
   if (enable_source_maps) {
@@ -234,6 +235,7 @@ template("build_webui") {
       if (defined(invoker.css_files)) {
         in_files += invoker.css_files
       }
+      if (defined(invoker.exclude_html_css_preprocess_files)) { in_files -= invoker.exclude_html_css_preprocess_files }
     }
   }
 
@@ -260,6 +262,7 @@ template("build_webui") {
           }
         }
       }
+      if (defined(invoker.preprocess_deps)) { public_deps = invoker.preprocess_deps }
     }
   }
 
