diff --git a/ui/webui/resources/BUILD.gn b/ui/webui/resources/BUILD.gn
index 3693085d41aa6610b8f318748f33adfa1660e87b..aa6afe1861d713de46bb2f1bf80aff6e9ca36879 100644
--- a/ui/webui/resources/BUILD.gn
+++ b/ui/webui/resources/BUILD.gn
@@ -126,6 +126,7 @@ generate_grd("build_grd") {
       "$root_gen_dir/ui/webui/resources/cr_components/page_image_service/resources.grdp",
     ]
   }
+  import("//brave/ui/webui/resources/sources.gni") grdp_files += brave_resources_extra_grdps public_deps += brave_resources_extra_grdps_deps
 }
 
 if (is_chromeos_ash) {
