diff --git a/ui/webui/resources/cr_elements/BUILD.gn b/ui/webui/resources/cr_elements/BUILD.gn
index 0c896b2c01e26c6955f6e18202badaf22484f8fa..20e262a59c91abbbb53efc13e6a1e87aa67f89cd 100644
--- a/ui/webui/resources/cr_elements/BUILD.gn
+++ b/ui/webui/resources/cr_elements/BUILD.gn
@@ -178,6 +178,7 @@ build_webui("build") {
   ts_composite = true
   ts_deps = [ "../js:build_ts" ]
   if (include_polymer) {
+    ts_deps += ["//brave/ui/webui/resources:build_ts"]
     ts_deps += [
       "//third_party/lit/v3_0:build_ts",
       "//third_party/polymer/v3_0:library",
