diff --git a/BUILD.gn b/BUILD.gn
index 74b40b55bd816e6d9a8cbccbfa5e8415ff2d986c..5b91722df67b21712c63872699eb35be05666393 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -747,6 +747,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3109,6 +3110,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
