diff --git a/BUILD.gn b/BUILD.gn
index d75db0338a7b340f05d86ae75c192bb11661ad8f..4bd0e2d6b71308b6adc5fc85012f966799f3f312 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -747,6 +747,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3088,6 +3089,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
